exercises/book-store/src/example/java/Bookstore.java

```diff
@@ -0,0 +1,64 @@
+import java.awt.print.Printable;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Bookstore{
+
+	private int bookPrice, maxGroupSize;
+	private double discountIncrement;
+	private List<Integer> books;
+	private static double[] discount_tiers = {0,5,10,20,25};   	
```

Comment:
Normally a constant would be `ALL_CAPITALS` like this.  Most editors will recognize them as such and make them easier to spot in the code.